packages/dependency-extraction-webpack-plugin/README.md

```diff
@@ -27,16 +27,38 @@ Use this plugin as you would other webpack plugins:
 
 ```js
 // webpack.config.js
-const WordPressExternalDependenciesPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
 
 module.exports = {
   // …snip
   plugins: [
-    new WordPressExternalDependenciesPlugin(),
+    new DependencyExtractionWebpackPlugin(),
   ]
 }
 ```
 
+**Note:** Multiple instances of the plugin are not supported and may produced unexpected results. If
+you plan to extend the webpack configuration from `@wordpress/scripts` with your own `DependencyExtractionWebpackPlugin`, be sure to
+remove the default instance of the plugin:
+
+```js
+const defaultConfig = require('@wordpress/scripts/config/webpack.config');
```

Comment:
Nit: this example should follow WordPress coding styles in regards to putting spaces around almost everything :)

Reply:
Cleaned it up, thanks